nitrite.go

```diff
@@ -264,7 +290,7 @@ func Verify(data []byte, options VerifyOptions) (*Result, error) {
 		return nil, ErrMandatoryFieldsMissing
 	}
 
-	if !options.AllowSelfSignedCert && nil == doc.CABundle {
+	if nil == doc.CABundle {
```

Comment:
While changing the interface doesn't cause too much of an issue, removing this feature (pretty much) means that we will have to prioritize porting sequencer over to newman.  Let's discuss.

Reply:
adding feature back in so we should be able to still support nitriding for a little longer.

Reply:
done